`no_panic_if` not exposed from `flux_attrs`

I found this while trying to write `no_panic` specs for vtock.

If we try to write a `no_panic_if` annotation on top of a function [in an external crate](https://github.com/ninehusky/my-minimal-repro), we get the following error:

```
Compiling my_minimal_repro v0.1.0 (/Users/acheung/Desktop/my_minimal_repro)
error[E0433]: failed to resolve: could not find `no_panic_if` in `flux_rs`
 --> src/main.rs:1:12
  |
1 | #[flux_rs::no_panic_if(true)]
  |            ^^^^^^^^^^^ could not find `no_panic_if` in `flux_rs`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `my_minimal_repro` (bin "my_minimal_repro") due to 1 previous error
```

--- a/lib/flux-attrs/src/lib.rs
+++ b/lib/flux-attrs/src/lib.rs
@@ -120,6 +120,11 @@ pub fn no_panic(attrs: TokenStream, tokens: TokenStream) -> TokenStream {
 }
 
 #[proc_macro_attribute]
+pub fn no_panic_if(attrs: TokenStream, tokens: TokenStream) -> TokenStream {
+    attr_impl::no_panic_if(attrs, tokens)
+}
+
+#[proc_macro_attribute]
 pub fn reft(attrs: TokenStream, tokens: TokenStream) -> TokenStream {
     attr_impl::reft(attrs, tokens)
 }
@@ -171,6 +176,7 @@ mod attr_sysroot {
         should_fail,
         reft,
         no_panic,
+        no_panic_if,
     );
 }
 
@@ -220,6 +226,7 @@ mod attr_dummy {
         ignore,
         should_fail,
         no_panic,
+        no_panic_if,
         reft,
     );
 }

